modules/base/files/logsalmsg

```diff
@@ -7,4 +7,9 @@ else
   ACTOR=$USER
 fi
 
+if [ $ACTOR = '' ]
```

Comment:
Why do this instead of just `|| [ $USER = 'www-data' ]` above?

Reply:
Unless my logic is off this wouldn't work anyways as this exact setting would be the default as matching the above statement anyways.